src/quo/components/tags/context_tag/view.cljs

```diff
@@ -81,95 +83,100 @@
        context]]]))
 
 (defn- view-internal
-  [{:keys [type size state blur? customization-color profile-picture full-name users
+  [{:keys [type theme size state blur? customization-color profile-picture full-name users
            group-name amount token network-logo network-name networks
-           account-name emoji collectible collectible-name collectible-number duration container-style]
+           account-name emoji collectible collectible-name collectible-number
+           dapp-name dapp-logo duration container-style]
     :or   {customization-color :blue
            type                :default
-           state               :default}
+           state               :default
+           theme               (quo.theme/use-theme)}
```

Comment:
I wonder if it's ok to use the theme hook here and not in the function body. Does this count as the function body?

Reply:
it's basically a wrapper for `React.useContext` so it works exactly the same if placed here or in function body